docs: restructure and add more content (#2232)
caos/zitadel: docs/docs/guides/customization/texts.md

---
title: Customized Texts
---

> You are able to customize some of the texts in ZITADEL.

Reviewer: ```suggestion
You are able to customize the texts used from ZITADEL.
```
